modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -40,6 +40,8 @@ private function __construct() {
 			add_filter( 'admin_body_class', array( $this, 'add_iframed_body_class' ) );
 		}
 
+		add_filter( 'jetpack_auth_cookie_samesite', array( $this, 'set_samesite_auth_cookie' ) );
```

Comment:
I wonder if this could help fixing some of the issues found in p1HpG7-8gO-p2. In that case, we might want to add this to a more generic module rather than the `wpcom-block-editor` one.